functions/airtableTriager.js

```diff
@@ -29,54 +120,124 @@ async function onNewIntake(id, fields) {
 
     const neighborhoodChanID = CHANNEL_TO_ID[neighborhoodChanName]
 
-    // TODO : add some better content
     const res = await bot.chat.postMessage({
         "channel": neighborhoodChanID,
-        "text": `HELLO! There is a new ticket: ${fields["Ticket ID"]}`,
+        "text": await getIntakePostContent(fields),
     })
 
+    // TODO : we should retry as we might be getting throttled
     if (!res["ok"]) {
-        console.error(`Encountered an error posting ticket to #${neighborhoodChanName}: ${id}`)
+        console.error(`Encountered an error posting ticket ${id} to #${neighborhoodChanName}: ${res["error"]}`)
         return
     }
+
+    meta["intakePostChan"] = neighborhoodChanID
+    meta["intakePostTs"] = res["ts"]
 }
 
-// Runs every minute
-exports.poll.airtableIntakeTickets = functions.pubsub.schedule('* * * * *').onRun(async () => {
-    const STATUS_TO_CBS = {
-        "Seeking Volunteer": [onNewIntake],
-        "Assigned / In Progress": [],
-        "Complete": [],
-        "Not Bed-Stuy": [],
+async function onIntakeAssigned(id, fields, meta) {
+    console.log(`onIntakeAssigned(${id})`)
+
+    if (!meta["intakePostChan"] || !meta["intakePostTs"]) {
+        console.error(`This ticket wasn't posted to slack: ${id}`)
+        return
+    }
+
+    let res = await bot.chat.update({
+        channel: meta["intakePostChan"],
+        ts: meta["intakePostTs"],
+        text: await getIntakePostContent(fields),
+    })
+
+    // TODO : we should retry as we might be getting throttled
+    if (!res["ok"]) {
+        console.error(`Encountered an error updating ticket ${id}: ${res["error"]}`)
+        return
     }
 
-    const changedTickets = await getChangedIntakeTickets()
+    meta["intakePostTs"] = res["ts"]
 
-    if (changedTickets.length === 0) {
-        return null
+    // TODO : use the delivery volunteer's id
+    res = await bot.chat.postMessage({
```

Comment:
instead of these both being called `res` the first one could be `intakePostResponse` and the second could be `deliveryMessageResponse` ?